scripts/bracket_orders.js

```diff
@@ -3,8 +3,8 @@ const { fetchTokenInfoFromExchange, getExchange, getSafe, buildOrders } = requir
   artifacts
 )
 const { isPriceReasonable, areBoundsReasonable } = require("./utils/price_utils.js")(web3, artifacts)
-const { signAndSend, promptUser } = require("./utils/sign_and_send")(web3, artifacts)
-const { proceedAnyways } = require("./utils/user_interface_helpers")
```

Comment:
I just realized this line should not be working in the first place since the file that is required exports a function and not a "generic" object. So why isn't destructuring failing here?

Reply:
I am not sure I understand the question/problem here.... the file `./utils/user_interface_helpers` exports `proceedAnyways`.

Reply:
Oh yes. I see what is happening here. Weird stuff that it was even working!